input.cc

```diff
@@ -233,25 +233,33 @@ static void read_phixs_data_table(FILE *phixsdata, const int nphixspoints_inputt
 }
 
 static void read_phixs_data(int phixs_file_version) {
-  globals::nbfcontinua_ground = 0;
-  globals::nbfcontinua = 0;
+  if (USE_TWO_PHIXS_FILES == false) {
+    globals::nbfcontinua_ground = 0;
+    globals::nbfcontinua = 0;
+  }
```

Comment:
Is this special case needed? It looks like you could set these to zero in read_atomicdata_files() just as you do when USE_TWO_PHIXS_FILES == true.